src/main/resources/application.properties

```diff
@@ -4,8 +4,10 @@ unblu.serverUrl=https://example.unblu.cloud
 unblu.entryPath=/unblu
 unblu.apiKey=MZsy5sFESYqU7MawXZgR_w
 unblu.publicKey=classpath:unblu.pub
+unblu.aes-encryption-key=JlFjDP9IHFv4Nho0Rk1lnPC/k7xh29uFImTyX0nDI5g=
```

Comment:
should we keep the naming convention and use `aesEncryptionKey`?

Reply:
done